src/Transports.AspNetCore/Extensions/GraphQLBuilderExtensions.cs

```diff
@@ -137,7 +137,6 @@ public static IGraphQLBuilder AddWebSocketAuthentication(this IGraphQLBuilder bu
     public static IGraphQLBuilder AddAuthorizationRule(this IGraphQLBuilder builder)
     {
         builder.AddValidationRule<AuthorizationValidationRule>(true);
-        builder.Services.TryRegister<IHttpContextAccessor, HttpContextAccessor>(DI.ServiceLifetime.Singleton);
```

Comment:
No more need to use `IHttpContextAccessor`

Reply:
👍